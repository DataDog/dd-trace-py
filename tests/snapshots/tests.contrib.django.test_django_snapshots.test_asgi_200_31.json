[[
  {
    "name": "django.request",
    "service": "django",
    "resource": "GET ^$",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "web",
    "meta": {
      "asgi.version": "3.0",
      "django.request.class": "django.core.handlers.asgi.ASGIRequest",
      "django.response.class": "django.http.response.HttpResponse",
      "django.user.is_authenticated": "False",
      "django.view": "tests.contrib.django.views.index",
      "http.method": "GET",
      "http.route": "^$",
      "http.status_code": "200",
      "http.url": "http://localhost:8000/",
      "http.version": "1.1",
      "runtime-id": "4bd10959dbea43a79d725b40f61e885f"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 3280
    },
    "duration": 8594000,
    "start": 1633585512684941000
  },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.contrib.sessions.middleware.SessionMiddleware.__call__",
       "trace_id": 0,
       "span_id": 2,
       "parent_id": 1,
       "duration": 3997000,
       "start": 1633585512685756000
     },
        {
          "name": "django.middleware",
          "service": "django",
          "resource": "django.contrib.sessions.middleware.SessionMiddleware.process_request",
          "trace_id": 0,
          "span_id": 3,
          "parent_id": 2,
          "duration": 82000,
          "start": 1633585512685803000
        },
        {
          "name": "django.middleware",
          "service": "django",
          "resource": "django.middleware.common.CommonMiddleware.__call__",
          "trace_id": 0,
          "span_id": 4,
          "parent_id": 2,
          "duration": 3724000,
          "start": 1633585512685933000
        },
           {
             "name": "django.middleware",
             "service": "django",
             "resource": "django.middleware.common.CommonMiddleware.process_request",
             "trace_id": 0,
             "span_id": 6,
             "parent_id": 4,
             "duration": 126000,
             "start": 1633585512685976000
           },
           {
             "name": "django.middleware",
             "service": "django",
             "resource": "django.middleware.csrf.CsrfViewMiddleware.__call__",
             "trace_id": 0,
             "span_id": 7,
             "parent_id": 4,
             "duration": 3409000,
             "start": 1633585512686147000
           },
              {
                "name": "django.middleware",
                "service": "django",
                "resource": "django.middleware.csrf.CsrfViewMiddleware.process_request",
                "trace_id": 0,
                "span_id": 9,
                "parent_id": 7,
                "duration": 26000,
                "start": 1633585512686190000
              },
              {
                "name": "django.middleware",
                "service": "django",
                "resource": "django.contrib.auth.middleware.AuthenticationMiddleware.__call__",
                "trace_id": 0,
                "span_id": 10,
                "parent_id": 7,
                "duration": 3204000,
                "start": 1633585512686258000
              },
                 {
                   "name": "django.middleware",
                   "service": "django",
                   "resource": "django.contrib.auth.middleware.AuthenticationMiddleware.process_request",
                   "trace_id": 0,
                   "span_id": 12,
                   "parent_id": 10,
                   "duration": 29000,
                   "start": 1633585512686298000
                 },
                 {
                   "name": "django.middleware",
                   "service": "django",
                   "resource": "django.contrib.messages.middleware.MessageMiddleware.__call__",
                   "trace_id": 0,
                   "span_id": 13,
                   "parent_id": 10,
                   "duration": 3059000,
                   "start": 1633585512686377000
                 },
                    {
                      "name": "django.middleware",
                      "service": "django",
                      "resource": "django.contrib.messages.middleware.MessageMiddleware.process_request",
                      "trace_id": 0,
                      "span_id": 14,
                      "parent_id": 13,
                      "duration": 93000,
                      "start": 1633585512686417000
                    },
                    {
                      "name": "django.middleware",
                      "service": "django",
                      "resource": "django.middleware.clickjacking.XFrameOptionsMiddleware.__call__",
                      "trace_id": 0,
                      "span_id": 15,
                      "parent_id": 13,
                      "duration": 2778000,
                      "start": 1633585512686551000
                    },
                       {
                         "name": "django.middleware",
                         "service": "django",
                         "resource": "django.middleware.security.SecurityMiddleware.__call__",
                         "trace_id": 0,
                         "span_id": 17,
                         "parent_id": 15,
                         "duration": 2502000,
                         "start": 1633585512686600000
                       },
                          {
                            "name": "django.middleware",
                            "service": "django",
                            "resource": "django.middleware.security.SecurityMiddleware.process_request",
                            "trace_id": 0,
                            "span_id": 19,
                            "parent_id": 17,
                            "duration": 26000,
                            "start": 1633585512686640000
                          },
                          {
                            "name": "django.middleware",
                            "service": "django",
                            "resource": "tests.contrib.django.middleware.ClsMiddleware.__call__",
                            "trace_id": 0,
                            "span_id": 20,
                            "parent_id": 17,
                            "duration": 2278000,
                            "start": 1633585512686706000
                          },
                             {
                               "name": "django.middleware",
                               "service": "django",
                               "resource": "tests.contrib.django.middleware.fn_middleware",
                               "trace_id": 0,
                               "span_id": 22,
                               "parent_id": 20,
                               "duration": 2212000,
                               "start": 1633585512686746000
                             },
                                {
                                  "name": "django.middleware",
                                  "service": "django",
                                  "resource": "tests.contrib.django.middleware.EverythingMiddleware.__call__",
                                  "trace_id": 0,
                                  "span_id": 23,
                                  "parent_id": 22,
                                  "duration": 2133000,
                                  "start": 1633585512686788000
                                },
                                   {
                                     "name": "django.middleware",
                                     "service": "django",
                                     "resource": "django.middleware.csrf.CsrfViewMiddleware.process_view",
                                     "trace_id": 0,
                                     "span_id": 24,
                                     "parent_id": 23,
                                     "duration": 32000,
                                     "start": 1633585512687547000
                                   },
                                   {
                                     "name": "django.middleware",
                                     "service": "django",
                                     "resource": "tests.contrib.django.middleware.EverythingMiddleware.process_view",
                                     "trace_id": 0,
                                     "span_id": 25,
                                     "parent_id": 23,
                                     "duration": 26000,
                                     "start": 1633585512687816000
                                   },
                                   {
                                     "name": "django.view",
                                     "service": "django",
                                     "resource": "tests.contrib.django.views.index",
                                     "trace_id": 0,
                                     "span_id": 26,
                                     "parent_id": 23,
                                     "duration": 64000,
                                     "start": 1633585512688305000
                                   },
                          {
                            "name": "django.middleware",
                            "service": "django",
                            "resource": "django.middleware.security.SecurityMiddleware.process_response",
                            "trace_id": 0,
                            "span_id": 21,
                            "parent_id": 17,
                            "duration": 46000,
                            "start": 1633585512689030000
                          },
                       {
                         "name": "django.middleware",
                         "service": "django",
                         "resource": "django.middleware.clickjacking.XFrameOptionsMiddleware.process_response",
                         "trace_id": 0,
                         "span_id": 18,
                         "parent_id": 15,
                         "duration": 139000,
                         "start": 1633585512689141000
                       },
                    {
                      "name": "django.middleware",
                      "service": "django",
                      "resource": "django.contrib.messages.middleware.MessageMiddleware.process_response",
                      "trace_id": 0,
                      "span_id": 16,
                      "parent_id": 13,
                      "duration": 32000,
                      "start": 1633585512689379000
                    },
              {
                "name": "django.middleware",
                "service": "django",
                "resource": "django.middleware.csrf.CsrfViewMiddleware.process_response",
                "trace_id": 0,
                "span_id": 11,
                "parent_id": 7,
                "duration": 29000,
                "start": 1633585512689503000
              },
           {
             "name": "django.middleware",
             "service": "django",
             "resource": "django.middleware.common.CommonMiddleware.process_response",
             "trace_id": 0,
             "span_id": 8,
             "parent_id": 4,
             "duration": 37000,
             "start": 1633585512689596000
           },
        {
          "name": "django.middleware",
          "service": "django",
          "resource": "django.contrib.sessions.middleware.SessionMiddleware.process_response",
          "trace_id": 0,
          "span_id": 5,
          "parent_id": 2,
          "duration": 32000,
          "start": 1633585512689697000
        }]]
