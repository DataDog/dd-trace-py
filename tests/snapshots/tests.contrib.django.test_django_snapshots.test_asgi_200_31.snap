[[{"name" "django.request"
   "service" "django"
   "resource" "GET ^$"
   "type" "web"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1628306009127982895
   "duration" 5578566
   "meta" {"runtime-id" "b8376213687e488bb16e55b26170f750"
           "http.route" "^$"
           "http.url" "http://localhost:8000/"
           "django.user.is_authenticated" "False"
           "http.status_code" "200"
           "http.method" "GET"
           "http.version" "1.1"
           "django.response.class" "django.http.response.HttpResponse"
           "django.request.class" "django.core.handlers.asgi.ASGIRequest"
           "django.view" "tests.contrib.django.views.index"
           "asgi.version" "3.0"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 674
              "_dd.tracer_kr" 1.0}}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.contrib.sessions.middleware.SessionMiddleware.__call__"
       "error" 0
       "span_id" 1
       "trace_id" 0
       "parent_id" 0
       "start" 1628306009128592734
       "duration" 2697101}
          {"name" "django.middleware"
           "service" "django"
           "resource" "django.contrib.sessions.middleware.SessionMiddleware.process_request"
           "error" 0
           "span_id" 2
           "trace_id" 0
           "parent_id" 1
           "start" 1628306009128624978
           "duration" 87973}
          {"name" "django.middleware"
           "service" "django"
           "resource" "django.middleware.common.CommonMiddleware.__call__"
           "error" 0
           "span_id" 3
           "trace_id" 0
           "parent_id" 1
           "start" 1628306009128737916
           "duration" 2477512}
              {"name" "django.middleware"
               "service" "django"
               "resource" "django.middleware.common.CommonMiddleware.process_request"
               "error" 0
               "span_id" 5
               "trace_id" 0
               "parent_id" 3
               "start" 1628306009128761844
               "duration" 80973}
              {"name" "django.middleware"
               "service" "django"
               "resource" "django.middleware.csrf.CsrfViewMiddleware.__call__"
               "error" 0
               "span_id" 6
               "trace_id" 0
               "parent_id" 3
               "start" 1628306009128866389
               "duration" 2311097}
                  {"name" "django.middleware"
                   "service" "django"
                   "resource" "django.middleware.csrf.CsrfViewMiddleware.process_request"
                   "error" 0
                   "span_id" 8
                   "trace_id" 0
                   "parent_id" 6
                   "start" 1628306009128893386
                   "duration" 49664}
                  {"name" "django.middleware"
                   "service" "django"
                   "resource" "django.contrib.auth.middleware.AuthenticationMiddleware.__call__"
                   "error" 0
                   "span_id" 9
                   "trace_id" 0
                   "parent_id" 6
                   "start" 1628306009128966008
                   "duration" 2178823}
                      {"name" "django.middleware"
                       "service" "django"
                       "resource" "django.contrib.auth.middleware.AuthenticationMiddleware.process_request"
                       "error" 0
                       "span_id" 11
                       "trace_id" 0
                       "parent_id" 9
                       "start" 1628306009128988970
                       "duration" 12868}
                      {"name" "django.middleware"
                       "service" "django"
                       "resource" "django.contrib.messages.middleware.MessageMiddleware.__call__"
                       "error" 0
                       "span_id" 12
                       "trace_id" 0
                       "parent_id" 9
                       "start" 1628306009129018976
                       "duration" 2119140}
                          {"name" "django.middleware"
                           "service" "django"
                           "resource" "django.contrib.messages.middleware.MessageMiddleware.process_request"
                           "error" 0
                           "span_id" 13
                           "trace_id" 0
                           "parent_id" 12
                           "start" 1628306009129081669
                           "duration" 114436}
                          {"name" "django.middleware"
                           "service" "django"
                           "resource" "django.middleware.clickjacking.XFrameOptionsMiddleware.__call__"
                           "error" 0
                           "span_id" 14
                           "trace_id" 0
                           "parent_id" 12
                           "start" 1628306009129217847
                           "duration" 1886152}
                              {"name" "django.middleware"
                               "service" "django"
                               "resource" "django.middleware.security.SecurityMiddleware.__call__"
                               "error" 0
                               "span_id" 16
                               "trace_id" 0
                               "parent_id" 14
                               "start" 1628306009129241200
                               "duration" 1822798}
                                  {"name" "django.middleware"
                                   "service" "django"
                                   "resource" "django.middleware.security.SecurityMiddleware.process_request"
                                   "error" 0
                                   "span_id" 18
                                   "trace_id" 0
                                   "parent_id" 16
                                   "start" 1628306009129265328
                                   "duration" 10006}
                                  {"name" "django.middleware"
                                   "service" "django"
                                   "resource" "tests.contrib.django.middleware.ClsMiddleware.__call__"
                                   "error" 0
                                   "span_id" 19
                                   "trace_id" 0
                                   "parent_id" 16
                                   "start" 1628306009129292075
                                   "duration" 1720934}
                                      {"name" "django.middleware"
                                       "service" "django"
                                       "resource" "tests.contrib.django.middleware.EverythingMiddleware"
                                       "error" 0
                                       "span_id" 21
                                       "trace_id" 0
                                       "parent_id" 19
                                       "start" 1628306009129309987
                                       "duration" 1696019}
                                          {"name" "django.middleware"
                                           "service" "django"
                                           "resource" "tests.contrib.django.middleware.EverythingMiddleware.__call__"
                                           "error" 0
                                           "span_id" 22
                                           "trace_id" 0
                                           "parent_id" 21
                                           "start" 1628306009129327666
                                           "duration" 1667342}
                                              {"name" "django.middleware"
                                               "service" "django"
                                               "resource" "django.middleware.csrf.CsrfViewMiddleware.process_view"
                                               "error" 0
                                               "span_id" 23
                                               "trace_id" 0
                                               "parent_id" 22
                                               "start" 1628306009129980589
                                               "duration" 19248}
                                              {"name" "django.middleware"
                                               "service" "django"
                                               "resource" "tests.contrib.django.middleware.EverythingMiddleware.process_view"
                                               "error" 0
                                               "span_id" 24
                                               "trace_id" 0
                                               "parent_id" 22
                                               "start" 1628306009130206992
                                               "duration" 11881}
                                              {"name" "django.view"
                                               "service" "django"
                                               "resource" "tests.contrib.django.views.index"
                                               "error" 0
                                               "span_id" 25
                                               "trace_id" 0
                                               "parent_id" 22
                                               "start" 1628306009130628284
                                               "duration" 41349}
                                  {"name" "django.middleware"
                                   "service" "django"
                                   "resource" "django.middleware.security.SecurityMiddleware.process_response"
                                   "error" 0
                                   "span_id" 20
                                   "trace_id" 0
                                   "parent_id" 16
                                   "start" 1628306009131033421
                                   "duration" 22794}
                              {"name" "django.middleware"
                               "service" "django"
                               "resource" "django.middleware.clickjacking.XFrameOptionsMiddleware.process_response"
                               "error" 0
                               "span_id" 17
                               "trace_id" 0
                               "parent_id" 14
                               "start" 1628306009131081951
                               "duration" 14656}
                          {"name" "django.middleware"
                           "service" "django"
                           "resource" "django.contrib.messages.middleware.MessageMiddleware.process_response"
                           "error" 0
                           "span_id" 15
                           "trace_id" 0
                           "parent_id" 12
                           "start" 1628306009131120507
                           "duration" 10640}
                  {"name" "django.middleware"
                   "service" "django"
                   "resource" "django.middleware.csrf.CsrfViewMiddleware.process_response"
                   "error" 0
                   "span_id" 10
                   "trace_id" 0
                   "parent_id" 6
                   "start" 1628306009131161024
                   "duration" 9578}
              {"name" "django.middleware"
               "service" "django"
               "resource" "django.middleware.common.CommonMiddleware.process_response"
               "error" 0
               "span_id" 7
               "trace_id" 0
               "parent_id" 3
               "start" 1628306009131193674
               "duration" 14625}
          {"name" "django.middleware"
           "service" "django"
           "resource" "django.contrib.sessions.middleware.SessionMiddleware.process_response"
           "error" 0
           "span_id" 4
           "trace_id" 0
           "parent_id" 1
           "start" 1628306009131265501
           "duration" 16242}]]
